modules/serverless-autoscaling-dashboard-navigating.adoc

```diff
@@ -0,0 +1,8 @@
+[id="serverless-autoscaling-dashboard-navigating_{context}"]
+= Navigating to the autoscaling dashboard
+
+.Procedure
+
+. In the {product-title} Web UI, go to the *Monitoring -> Dashboards* page.
```

Comment:
Here, shouldn't it be "Observe -> Dashboards" page instead or "Monitoring -> Dashboards" ?

Reply:
I believe it changed from Monitoring to Observe in 4.9, please verify and update. You may need to open a separate PR to change only 4.9 and 4.10 to Observe and keep 4.6-4.8 as Monitoring.

Reply:
ACK, will do it as a separate PR. Thanks.